api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -686,6 +686,85 @@ private String getQualifiedColumnName(Domain currentDomain, String columnName) {
+            + "        person_ids = person_ids.union(dataset['PERSON_ID'])\n"
+            + "    elif 'person_id' in dataset:\n"
+            + "        person_ids = person_ids.union(dataset['person_id']) \n"
+            + "\n\n"
+            + "with open('cohort_sample_names.txt', 'w') as cohort_file:\n"
+            + "    for person_id in person_ids:\n"
+            + "        cohort_file.write(str(person_id) + '\\n')\n"
+            + "    cohort_file.close()\n",
+        "%%bash\n\n"
+            + "uuid=$(cat /proc/sys/kernel/random/uuid | sed s/-/_/g)\n"
+            + "EXPORT_TABLE=\"fc-aou-cdr-synth-test.tmp_shared_cohort_extract.${uuid}\"\n"
+            + "\n"
+            + "python3 /genomics/microarray/raw_array_cohort_extract.py \\\n"
+            + "          --dataset fc-aou-cdr-synth-test.microarray_data \\\n"
+            + "          --fq_destination_table ${EXPORT_TABLE} \\\n"
+            + "          --query_project "
+            + dbWorkspace.getWorkspaceNamespace()
+            + " \\\n"
+            + "          --sample_mapping_table fc-aou-cdr-synth-test.microarray_data.sample_list \\\n"
+            + "          --cohort_sample_names_file cohort_sample_names.txt \\\n"
+            + "          --sample_map_outfile cohort_sample_map.csv\n"
+            + "\n"
+            + "gatk ArrayExtractCohort \\\n"
+            + "        -R /genomics/Homo_sapiens_assembly19.fasta \\\n"
+            + "        -O cohort.vcf \\\n"
+            + "        --probe-info-csv /genomics/microarray/probe_info.csv \\\n"
+            + "        --project-id fc-aou-cdr-synth-test \\\n"
+            + "        --cohort-sample-file cohort_sample_map.csv \\\n"
+            + "        --use-compressed-data \"false\" \\\n"
+            + "        --cohort-extract-table ${EXPORT_TABLE} \\\n"
+            + "        --local-sort-max-records-in-ram \"1000000\"");
+  }
+
+  @Override
+  public List<String> generatePlinkDemoCode() {
+    return ImmutableList.of(
+        "import random\n\n"
+            + "phenotypes_table = []\n"
+            + "for person_id in person_ids:\n"
+            + "    family_id = 0\n"
+            + "    person_id = person_id\n"
+            + "    phenotype_1 = random.randint(0, 2) # Change this value to what makes sense for your research by looking through the dataset(s)\n"
+            + "    phenotype_2 = random.randint(0, 2) # Change this value as well or remove if you are only processing one phenotype \n"
+            + "    phenotypes_table.append([family_id, person_id, phenotype_1, phenotype_2])\n"
+            + "\n"
+            + "cohort_phenotypes = pandas.DataFrame(phenotypes_table) \n"
+            + "cohort_phenotypes.to_csv('phenotypes.phe', header=False, index=False, sep=' ')",
+        "%%bash\n\n"
+            + "plink --vcf-half-call m --const-fid 0 --vcf cohort.vcf\n"
+            + "plink --bfile plink --pheno phenotypes.phe --all-pheno --allow-no-sex --assoc --out results\n"
```

Comment:
I guess `plink` is the default filename, but I find this pretty confusing. I would expect this to be called something like `cohort.bed` or similar; wdyt about specifying the output filename here?